case-lib/lib.sh

```diff
@@ -109,7 +109,10 @@ func_lib_start_log_collect()
 {
     local is_etrace=${1:-0} ldcFile
 
-    ldcFile=$(find_ldc_file) || return $?
+    ldcFile=$(find_ldc_file) || {
+       dlogw "No .ldc file found, firmware log will not be collected"
+       return 0
+    }
```

Comment:
this test is a bit odd. In itself, it looks fine, but if you scroll down the code there are still errors that are reported if SOFLOGGER is not defined, or not well configured.

The ask is really to completely avoid depending on sof-logger and ONLY use alsa-lib/alsa-bat for test results.